example/README.md

```diff
@@ -118,7 +118,7 @@ PRODUCT_TO_CPE_MAPPING = {
 </pre>
 <pre>
 ...
-MULTI_PLATFORM_LIST = ["rhel", "fedora", "rhosp", "rhv", "debian", "ubuntu",
+MULTI_PLATFORM_LIST = ["rhel", "fedora", "rhosp", "rhv", "debian",
```

Comment:
I assume this `ubuntu` means any version?

Reply:
fixed